include/functions.h

```diff
@@ -4020,7 +4027,7 @@ void func_801A7794(u32 param_1, u32 param_2, u32 param_3);
 // void func_801A7B10(void);
 // void func_801A7D04(void);
 // void func_801A7D84(void);
-// void func_801A89A8(void);
+// void func_801A89A8(UNK_TYPE arg0);
```

Comment:
isn't this just s32?

Reply:
I looked at the generated code of mips2c of this function and it looked like that, but I'm not completely sure. It could be a pointer or something else. The only thing I'm sure is it only takes 1 parameter.